utils/python/CIME/XML/build.py

```diff
@@ -1,9 +1,9 @@
 """
```

Comment:
This was requested by @jedwards4b, based on the fact that this is the file responsible for reading `config_build.xml`.